[2024-03-09T03:00:18.930+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.load_data_to_postgres scheduled__2024-03-09T02:00:00+00:00 [queued]>
[2024-03-09T03:00:18.940+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.load_data_to_postgres scheduled__2024-03-09T02:00:00+00:00 [queued]>
[2024-03-09T03:00:18.941+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-03-09T03:00:18.970+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): load_data_to_postgres> on 2024-03-09 02:00:00+00:00
[2024-03-09T03:00:18.980+0000] {standard_task_runner.py:60} INFO - Started process 3231 to run task
[2024-03-09T03:00:18.983+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'load_data_to_postgres', 'scheduled__2024-03-09T02:00:00+00:00', '--job-id', '461', '--raw', '--subdir', 'DAGS_FOLDER/weather_api__dag.py', '--cfg-path', '/tmp/tmp_waya84b']
[2024-03-09T03:00:18.984+0000] {standard_task_runner.py:88} INFO - Job 461: Subtask load_data_to_postgres
[2024-03-09T03:00:19.027+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.load_data_to_postgres scheduled__2024-03-09T02:00:00+00:00 [running]> on host 70ec329f3d65
[2024-03-09T03:00:19.114+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='kan@odds.team' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='load_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-03-09T02:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-03-09T02:00:00+00:00'
[2024-03-09T03:00:19.124+0000] {logging_mixin.py:188} INFO - /opt/***/dags/weather_data_2024-03-09T02:00:00+00:00.json
[2024-03-09T03:00:19.125+0000] {warnings.py:109} WARNING - /opt/***/dags/weather_api__dag.py:88: AirflowProviderDeprecationWarning: The "schema" arg has been renamed to "database" as it contained the database name.Please use "database" to set the database name.
  pg_hook = PostgresHook(

[2024-03-09T03:00:19.133+0000] {base.py:83} INFO - Using connection ID 'my_***_conn' for task execution.
[2024-03-09T03:00:19.146+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-03-09T03:00:19.156+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=load_data_to_***, execution_date=20240309T020000, start_date=20240309T030018, end_date=20240309T030019
[2024-03-09T03:00:19.197+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-09T03:00:19.213+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
